.travis.yml

```diff
@@ -1,6 +1,15 @@
 os: linux
-dist: trusty
+dist: xenial
 language: python
+sudo: required
+cache:
+  - apt
+env:
+  global:
+    # Adapted from https://github.com/felixduvallet/ros-travis-integration/blob/master/.travis.yml
+    - CI_SOURCE_PATH=$(pwd)
```

Comment:
Is `CI_SOURCE_PATH` just the root of the repo? Can I suggest to change it to `REPO_ROOT` in that case?

Reply:
done